### ISSUE TITLE
Adding library item with comment loses comment

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#389]
I entered a comment when adding a Library Item reply type (i.e. reply, click From Library tab, select a library item, add comment, press the (tiny) Add To Thread button), at:

http://camcycle.cyclescape.org/threads/157#message_24285

but the comment I included got lost.

I don't know if this is a view rendering issue and that the data has actually been stored, or whether the database has actually captured the comment.

**Comment by smsm1** [comment]
This is similar to or the same as #243

**Comment by mvl22** [comment]
@smsm1 No, this is a different issue. I didn't switch tabs at all. That issue basically refers to the UI problem; this is about actual data loss upon a normal submission.

**Comment by nikolai-b** [comment]
I spent a while working out what was meant to happen when I clicked select in the library items tab (for me it did nothing).  Was the selecting the library items part working?  I've fixed it for my chrome and ff in https://github.com/cyclestreets/cyclescape/commit/ce21c276ab6afad8d56f76eee7860537bca14245 (on staging) but if it is working on your browsers in production I was hesitant to push this to master.

Anyway, back the the submitted bug: that is now fixed.
